cmd/maya-exporter/app/command/exporter_test.go

```diff
@@ -77,3 +81,38 @@ func startTestServer(t *testing.T, options *VolumeExporterOptions, errMsg chan e
 		errMsg <- options.StartMayaExporter()
 	}()
 }
+
+func TestJsonHandler(t *testing.T) {
+	cases := map[string]struct {
+		targetURL string
+		err       error
+	}{
+		"When URL is correct": {
+			targetURL: "http://localhost:9500" + metricsPath + "json/",
+			err:       nil,
+		},
```

Comment:
can we add few more test cases?

Reply:
I think it will be difficult as the function we are using for gathering information is from Prometheus. It will be difficult to mock it.